lib/transaction.js

```diff
@@ -122,6 +125,33 @@ module.exports = function transaction() {
   });
 };
 
+transaction.start = function start(modelBase) {
+  if (!modelBase) {
+    return Promise.reject(new Error('objection.transaction.start: provide at least one Model class to bind to the transaction'))
+  }
+
+  if (!utils.isSubclassOf(modelBase, Model)) {
+    return Promise.reject(new Error('objection.transaction.start: all but the last argument should be Model subclasses'));
+  }
+
+  var knex = modelBase.knex();
+
+  return new Promise(function(resolve, reject) {
+    knex.transaction(function (trx) {
+        modelBase = modelBase.knex(trx);
+
+      resolve({
```

Comment:
You resolve the promise with a custom object. You cannot bind models to this object using `bindKnex`. You can just resolve the promise with the `trx` object which is itself a knex instance.